### PR TITLE
Feature/helm chart

### PR DESCRIPTION
**Pull request by dblane-digicatapult**

### CODE REVIEW

File: helm/vitalam-ipfs/values.yaml [hunk 1]
```diff
@@ -1,7 +1,27 @@
...
+  # nodeHost: "" # This overrides vitalamNode.enabled
+  # nodePort:
+  logLevel: trace
+  ipfsApiPort: 5001
+  ipfsSwarmPort: 4001
+  ipfsDataPath: "/ipfs"
+  ipfsCommand: "/usr/local/bin/ipfs"
+  ipfsArgs:
+    - daemon
+  # ipfsBootNodeAddress: /dnsaddr/blah.test.com/p2p/PeerId # Valid Format for this is /multiaddr/PeerId see https://github.com/multiformats/multiaddr
```
Comment (mattdean-digicatapult) [inline helm/vitalam-ipfs/values.yaml:13]: this should probably be an array

File: helm/vitalam-ipfs/templates/statefulset.yaml [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+                name: {{ include "vitalam-ipfs.fullname" . }}-config
+          command: ['/bin/sh', '-c', '-e', '-x']
+          args:
+            - |
+              [ ! -f "$IPFS_PATH/config" ] && $ipfsCommand init --profile server;
+              $ipfsCommand config Addresses.API /ip4/0.0.0.0/tcp/$ipfsApiPort;
+              $ipfsCommand config --json Addresses.Swarm "[\"/ip4/0.0.0.0/tcp/$ipfsSwarmPort\"]";
+              $ipfsCommand config Datastore.StorageMax $(echo "$dataVolumeSize""Gb");
+              $ipfsCommand config --json API.HTTPHeaders.Access-Control-Allow-Origin "[\"http://0.0.0.0:$ipfsApiPort\", \"http://127.0.0.1:$ipfsApiPort\"]";
+              $ipfsCommand bootstrap rm --all;
```
Comment (mattdean-digicatapult) [inline helm/vitalam-ipfs/templates/statefulset.yaml:39]: There's probably more init we want avaialble but for now I'm happy to leave it

**Review by mattdean-digicatapult — CHANGES_REQUESTED** [review]

File: helm/vitalam-ipfs/values.yaml (hunk 1 above)
Comment (dblane-digicatapult) [reply helm/vitalam-ipfs/values.yaml:13]: Setting this as an array would mean doing some horrible looping in the `statefulSet` init container.  Happy to raise this as an Issue to be fixed at a later date.

Comment (mattdean-digicatapult) [reply helm/vitalam-ipfs/values.yaml:13]: Yeah ok that's a bit of a pain. Add an issue and we'll get it in

**Review by mattdean-digicatapult — APPROVED** [review]

**Pull request merged**